tigerpath/majors_and_certificates/majors/GEO.json

```diff
@@ -449,7 +449,8 @@
                   "explanation": "Five elective 300/400 level geosciences courses.",
                   "course_list": [
                     "GEO 3*",
-                    "GEO 4*"
+                    "GEO 4*",
+                    "GEO 202"

```

Comment:
This change is a typo

Reply:
I'm a little confused... if it's a typo, can you remove it?

Reply:
Fixed